platform/app/cypress.config.ts

```diff
@@ -33,5 +33,9 @@ export default defineConfig({
       mochaFile: 'cypress/results/test-output.xml',
       toConsole: true,
     },
+
+    // We go back two levels up to the repository root folder
+    videosFolder: '../../cypress/videos',
```

Comment:
I think it might be best  to either not commit this OR place the videos and screenshots in the ../cypress/results directory and add the appropriate .gitignore entries since we don't want those types of things in git. Thanks.